capsules/object_speed/meta.conf

```diff
@@ -0,0 +1,2 @@
+[about]
+api_compatibility_version = 0.0
```

Comment:
I wonder if there are any linters for new lines at ends of files?

Reply:
Added the newline